NOnion/Utility/BigIntegerExtensions.fs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NOnion.Utility
```

Comment:
why this commit doesnt remove the C# equivalent? and, wait, pls no type extensions, use static functions